website/docs/cli/commands/version.mdx

```diff
@@ -20,7 +20,18 @@ Execute the `atmos version` command like this:
 atmos version
 ```
 
-This will show the CLI version.
+This will show the CLI version. 
+
+From time to time, Atmos will check for updates. The frequency is configured in the `atmos.yaml`. There are six frequency options: minute, hourly, daily, weekly, monthly, and yearly. The default is to check `daily`, and if any unsupported values are passed this default will be used.
+
+It is also possible to turn off version checks in `atmos.yaml` by switching `enable` to `false`, or by setting the `ATMOS_VERSION_CHECK_ENABLED` environment variable to `false`, which overriders the config's settings. 
+
+```shell
+atmos version --check
+```
+
+The version command supports a `--check` flag. 
+It will force atmos to check for a new version, irrespective of the configuration settings.  
```

Comment:
```suggestion
This will force Atmos to check for a new version, irrespective of the configuration settings.  
```
```suggestion
It will force atmos to check for a new version, irrespective of the configuration settings.  
```

Reply:
Done